bioimageio/core/predict.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import sys
+
+import numpy as np
+import torch.cuda
+import xarray as xr
+
+from bioimageio.core.prediction_pipeline import create_prediction_pipeline
+from bioimageio.spec import import_package
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-m", "--model", help="bioimage model package (zip file)", required=True)
+parser.add_argument("image", nargs="?", help="image to process (.npy)")
+parser.add_argument("-o", "--output", nargs="?", help="output image (.npy)", required=True)
```

Comment:
would be nice expose specifying  device.

Reply:
Next PR